mdaviz/mda_file_viz.py

```diff
@@ -44,26 +44,50 @@ def setMetadata(self, text, *args, **kwargs):
         # tab=self.metadataPage
         self.metadata.setText(text)
 
-    def setData(self, text, *args, **kwargs):
-        self.data.setText(text)
-
     def setPlot(self, plot_widget):
         layout = self.plotPageMpl.layout()
-        utils.removeAllLayoutWidgets(
-            layout
-        )  # TODO replace with mainWindow.clearContent?
+        # TODO - question: # why are we removing all the layouts? What are we doing here?
+        # TODO - later: replace removeAllLayoutWidgets with mainWindow.clearContent?
+        # or clearContents(plot=True, data=False, metadata=False)
+        utils.removeAllLayoutWidgets(layout)
         layout.addWidget(plot_widget)
-        self.tabWidget.setCurrentWidget(self.plotPageMpl)
 
-    def isPlotBlank(self):
-        layout = self.plotPageMpl.layout()
-        if layout.count() == 0:
-            return True
-        plot_widget = layout.itemAt(0).widget()
-        # Check if the plot widget is an instance of chartView and has data items
-        if isinstance(plot_widget, ChartView):
-            return not plot_widget.hasDataItems()
-        return True  # If not a chartView instance, consider it as blank
+    # # TODO: unused? I do not think we use this:
```

Comment:
Agreed.  `git grep isPlotBlank` shows no other usage.  Remove this commented code.